### DISCUSSION TITLE
Search interface should show the api call it generates

### DISCUSSION THREAD

**Discussion by sethchandler** [freelawproject/courtlistener#4394] [Ideas]
To help users understand the formatting and content of api calls, the search frontend (https://www.courtlistener.com/opinion/) should show the api call it generates. It could provide a tab that showed it as a CLI call, a CURL call, a Python request and perhaps one or two other common formats. I suspect this wouldn't be too hard as surely it is the case that the front end indeed generates exactly such an api call.

**Comment by mlissner** [comment]
> surely it is the case that the front end indeed generates exactly such an api call.

Would that that were true, but no. Internally we have all the power in the world and we're not yet API-driven. 

But in general, the documentation says: 

![image](https://github.com/user-attachments/assets/141bd105-54f8-47e4-bd84-7eba5642050e)

Is that not good enough or is that failing somehow?
